jarbas/chamber_of_deputies/views.py

```diff
@@ -30,6 +30,11 @@ def get(self, request):
         values = map(self.request.query_params.get, params)
         filters = {k: v for k, v in zip(params, values) if v}
 
+        # filter cpf or cnpj without dot and dash
+        hasCfpCnpj = self._bool_param('cnpj_cpf')
```

Comment:
[`hasCfpCnpj` is not a recommended variable name according to PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). Also there's a typo `Cfp` instead of `Cpf` ; )